itkwidgets/integrations/environment.py

```diff
@@ -0,0 +1,24 @@
+IN_JUPYTER_NOTEBOOK = False
+IN_JUPYTERLAB = False
+IN_JUPYTERLITE = False
+IN_AWS = False
```

Comment:
If there can only be 1 as true, an Enum would enforce that.
```
def findEnv():
  if something:
    return Env.JUPYTERLAB
  if anotherThing:
   return Env.JUPYTERLITE

env = findEnv()
```

Reply:
Updated, thanks!